reg_writes.txt

```diff
@@ -0,0 +1,2670 @@
+0xC022 0x1
```

Comment:
Is this needed? If it is needed, I don't think it should exist in the top level directory.

Reply:
Done